src/DynamoCoreWpf/Views/Preview/WatchTree.xaml.cs

```diff
@@ -34,6 +43,8 @@ public WatchTree(WatchViewModel vm)
             DataContext = vm;
             this.Loaded += WatchTree_Loaded;
             this.Unloaded += WatchTree_Unloaded;
+
+            _ctrlResetTimer.Tick += _ctrlResetTimer_Tick;
```

Comment:
We are using this pattern inside the DynamoView (`_workspaceResizeTimer.Tick += _resizeTimer_Tick;`) without unsubscribing, I assumed this is safe.

Reply:
Can we unsubscribe to it? If possible? Probably add a Dispose method?

Reply:
Of course. The Watchnode does not have a `Dispose`, but a similar thing is happening inside the Loaded/Unloaded handlers.

Reply:
@zeusongit I removed the timer altogether. Sporadic tests were failing, I don't know if that was due to a leak, but I was able to remove the timer while tinkering. Sadly, I cannot figure out a way to test for the `Keyboard.Modifiers` combination, as far as I can see it's impossible to set that programmatically. So I don't have a way of Unit testing that functionality. I don't know if we have a different framework for Integration tests that we use to simulate clicks and user interactions, but probably that would be the way to do it.

Reply:
Thanks @dnenov, I was iffy with the timer approach myself, so thanks for looking into it. I will ask if this is something AGT tests can cover.

Reply:
Definitely ^